src/main/java/io/github/thebusybiscuit/slimefun4/utils/NumberUtils.java

```diff
@@ -171,9 +176,31 @@ private NumberUtils() {}
         }
     }
 
-    public static @Nonnull String getTimeLeft(int seconds) {
+    public static @Nonnull String getTimeLeft(int ticksLeft) {
         String timeleft = "";
 
+        /**
+         * This is the amount calculated in addProgress(). 
+         * This will be the scale used for timer.
+         */
+        double timeScale = (TICK_RATE / 10.0d) * 1;
```

Comment:
Is the * 1 necessary here. it will always stay the same.

Reply:
You are correct this is not required. I removed it now.

Reply:
i was doubting my self for a tad. opened my calc and did 0.1*1

Reply:
lol